internal/util/projutil/project_util.go

```diff
@@ -52,6 +55,18 @@ func MustInProjectRoot() {
 	}
 }
 
+func MustGoProject(cmd *cobra.Command) {
+	t := GetOperatorType()
+	switch t {
+	case OperatorTypeGo:
+	default:
+		s := fmt.Sprintf("'%s' is meant for Go projects.\n", cmd.CommandPath())
+		s += fmt.Sprintf("Migrate your project to a hybrid structure before running this command for %s operators.\n", strings.Title(t))
```

Comment:
We can remove this message. We don't have instructions for a hybrid structure yet. So this won't be very clear.

We can add more details on what's missing and how to add it later once we have consensus on the steps and cmd needed to transition to a hybrid project.

Reply:
Or we can point out that they're missing `cmd/manager/main.go` for now. Since that's all that's needed to run these commands.